### ISSUE TITLE
Degrading possible?

### ISSUE THREAD

**Issue by ryanlecompte** [afeld/jeditable-rails#11]
Hello,

When using jeditable-rails, is there a fairly easy way to degrade properly in the case where Javascript isn't available? In other words, if JS isn't available, simply go back to a standard form / submit button for editing?

Thanks!

Ryan

**Comment by afeld** [comment]
No, there isn't built-in support for it - would you expect there to be a separate form and submit button for every field?  What element would all of your `html_options` get applied to?

If you would like to take a crack at it, I would be happy to merge it in, but I'm not sure what the ideal behavior would be.

**Comment by ryanlecompte** [comment]
Thanks for responding! I guess it would be ideal to degrade somehow to just
normal input fields and then a single "submit" button to submit all the
values. Not sure how that work, it seems like you'd need to detect whether
or not JS is available on the client to know to create the submit button or
not. I'll give it some more thought for my particular use case.

Feel free to close this pull request.

Thanks!
